libcontainer/seccomp/types.go

```diff
@@ -0,0 +1,48 @@
+package seccomp
```

Comment:
+1 to adding a default action and removing the whitelist bool.

other changes sound ok for now